ts/store/reducers/wallet/wallets.ts

```diff
@@ -354,50 +345,29 @@ const reducer = (
       const wallets = isOfType(getType(paymentUpdateWalletPsp.success), action)
         ? action.payload.wallets
         : action.payload;
-      const favouriteWallet = wallets.find(_ => _.favourite === true);
       const newState = {
         ...state,
         walletById: pot.some(toIndexed(wallets, _ => _.idWallet))
       };
-      return {
-        ...newState,
-        favoriteWalletId:
-          favouriteWallet !== undefined
-            ? pot.some(favouriteWallet.idWallet)
-            : pot.none
-      };
+      return { ...newState };
```

Comment:
You could directly return 
```suggestion
      return {
        ...state,
        walletById: pot.some(toIndexed(wallets, _ => _.idWallet))
      };
```